peerdb-api/endpoints/create-peer/clickhouse.mdx

```diff
@@ -1,17 +1,10 @@
-
-```http
-POST api/v1/peers/create
-```
-
-This endpoint is used to create a peer. Peers represent data stores between which you can move data using a mirror.
-
-### Peers in this documentation:
-1. [Postgres peer](#postgres-peer)
-2. [Clickhouse peer](#clickhouse-peer)
-
-## Postgres Peer
-Here is the request and response format for creating a Postgres peer. SSH tunneling is optional for peer creation.
-
+---
+title: "Create a ClickHouse peer"
+---
+You can create a Clickhouse peer using this endpoint. Here is the request and response format for creating a Clickhouse peer.
+Note that the Clickhouse peer uses an intermediary S3 stage under the hood for performance. The S3 fields related to this are optional, because:
+- For PeerDB Cloud, the S3 bucket is managed by PeerDB
+- For PeerDB OSS, a minio bucket is provided as part of the stack.
```

Comment:
There is a gotcha here, OSS minio needs to be exposed if CH isn't on the same machine

Reply:
I don't think we need to go in-depth into that in the API doc
Should do it in the setup doc